synapse/lib/json.py

```diff
@@ -217,7 +229,15 @@ def reqjsonsafe(item: Any, strict: bool = False) -> None:
     '''
     if strict:
         try:
-            orjson.dumps(item)
+            _dumps(item)
+
+        except s_exc.MustBeJsonSafe:
+            raise
+
+        except UnicodeEncodeError as exc:
+            mesg = str(exc)
+            raise s_exc.MustBeJsonSafe(mesg=mesg)
+
         except Exception as exc:
             raise s_exc.MustBeJsonSafe(mesg=exc.args[0])
```

Comment:
```suggestion
            raise s_exc.MustBeJsonSafe(mesg=f'{exc.__class__.__name__}: {exc}')
```

`exc.args[0]` may not always be populated on all exceptions; a lack of populated value would throw an IndexError in this handler.